flake.nix

```diff
@@ -86,11 +87,15 @@
 
           # should be similar to the installer in the Dockerfile
           installPhase = ''
-            mkdir -p $out/scripts
+            mkdir -p $out
+            # static files that can be served by reverse-proxies
+            cp -r apps/router/build/* $out
 
-            cp -r apps/router/build/* $out/
-            cp -r scripts/replace-react-env.js $out/scripts/replace-react-env.js
-            cp -r scripts/write-config-from-env.js $out/scripts/write-config-from-env.js
```

Comment:
It works for docker by replacing config.json with envs here: 
https://github.com/fedimint/ui/blob/25da5532737f7aa61a29a4ae394ae1bffc1fcd9b/Dockerfile#L35-L37

Removed these files as it doesn't make sense for NixOS (read-only filesystem). NixOS deploy generates it via nix, ref:
https://github.com/fedimint/nixos-deployment/blob/master/configuration.nix#L168-L176